### PR TITLE
chore: add version to v5 migration guide

### PR DESCRIPTION
**Pull request by peetzweg**
`ink!athon` version is not fixed yet to `0.7.0` but potentially is.

https://github.com/scio-labs/inkathon/pull/60

### CODE REVIEW

File: versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md [hunk 1]
```diff
@@ -122,9 +122,8 @@ Developer » Chain State » `contracts` » `palletVersion()` » Click on the `+`
 
 * Stable Rust >= 1.75
 * `cargo-contract` >= v4.0
-* `polkadot-js/api` and `polkadot-js/api-contract` >= TODO
-* `use-inkathon` >= TODO
-* ink!athon >= TODO
+* `polkadot-js/api` and `polkadot-js/api-contract` >= 10.12.1
+* ink!athon >= 0.7.0
```
Comment (cmichi) [inline versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md:126]: What about `use-inkathon`?

Comment (cmichi) [reply versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md:126]: Okay, can you add a note here that clarifies this to users?

Comment (peetzweg) [reply versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md:135]:
```
* `use-inkathon`, upgrade `polkadot-js/api` and `polkadot-js/api-contract` dependencies to >= 10.12.1
```

File: versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md [hunk 2]
```diff
@@ -131,8 +131,8 @@ Developer » Chain State » `contracts` » `palletVersion()` » Click on the `+`
 * `cargo-contract` >= v4.0
 * `substrate-contracts-node` >= 0.39.0
 * `polkadot-js/api` and `polkadot-js/api-contract` >= 10.12.1
-* `use-inkathon` >= TODO
-* ink!athon >= TODO
+* `use-inkathon`, upgrade `polkadot-js/api` and `polkadot-js/api-contract` dependencies to >= 10.12.1
```
Comment (cmichi) [inline versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md:134]:
```suggestion
* `use-inkathon`: upgrade the `polkadot-js/api` and `polkadot-js/api-contract` dependencies in your project to >= 10.12.1
```

**Review by cmichi — APPROVED** [review]
Thanks, Philip!

**Pull request merged**